frontend/src/mixins/validation/ValidatorMixin.ts

```diff
@@ -47,23 +49,23 @@ export default class ValidatorMixin extends Vue {
     if (!_.isNil(validationMessage)) {
       return validationMessage;
     }
-    if (infrastrukturabfrage.sobonRelevant === UncertainBoolean.True && 
-        _.isNil(infrastrukturabfrage.sobonJahr)) {
+    if (infrastrukturabfrage.sobonRelevant === UncertainBoolean.True &&
```

Comment:
Booleanoperatoren (hier &&) bei zeilenübergreifenden Operationen in der nächsten Zeile an den Anfang stellen.

Reply:
erledigt.